docs/examples/greeting.js

```diff
@@ -21,11 +21,11 @@ client.on('ready', () => {
 // Create an event listener for new guild members
 client.on('guildMemberAdd', member => {
   // Send the message to a designated channel on a server:
-  const channel = member.guild.channels.cache.find(ch => ch.name === 'member-log');
+  const channel = member.guild.channels.cache.find(channel => channel.name === 'member-log');
   // Do nothing if the channel wasn't found on this server
   if (!channel) return;
   // Send the message, mentioning the member
-  channel.send(`Welcome to the server, ${member}`);
+  channel.send(`Welcome to the server, ${member.displayName}`);
```

Comment:
Comment above specifically mentions mentioning the member, so this makes it incorrect.